### PR TITLE
plzip: init at 1.10

### PR DESCRIPTION
**Pull request by 360ied**
###### Description of changes

Plzip is a massively parallel (multi-threaded) implementation of lzip.

Homepage: https://www.nongnu.org/lzip/plzip.html

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/compression/plzip/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  buildInputs = [ lzlib ];
+
+  src = fetchurl {
+    url = "mirror://savannah/lzip/plzip/plzip-${version}.tar.lz";
+    sha256 = "62f16a67be0dabf0da7fd1cb7889fe5bfae3140cea6cafa1c39e7e35a5b3c661";
+  };
+
+  enableParallelBuilding = true;
+
+  meta = with lib; {
```
Comment (sikmir) [inline pkgs/tools/compression/plzip/default.nix:18]: What about `maintainers`?

Comment (360ied) [reply pkgs/tools/compression/plzip/default.nix:18]: i added myself to maintainers

File: pkgs/tools/compression/plzip/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  version = "1.10";
+  outputs = [ "out" "man" "info" ];
+
+  nativeBuildInputs = [ lzip texinfo ];
+  buildInputs = [ lzlib ];
+
+  src = fetchurl {
+    url = "mirror://savannah/lzip/plzip/plzip-${version}.tar.lz";
+    sha256 = "62f16a67be0dabf0da7fd1cb7889fe5bfae3140cea6cafa1c39e7e35a5b3c661";
+  };
```
Comment (SuperSandro2000) [inline pkgs/tools/compression/plzip/default.nix:14]:
```suggestion
  src = fetchurl {
    url = "mirror://savannah/lzip/plzip/plzip-${version}.tar.lz";
    sha256 = "62f16a67be0dabf0da7fd1cb7889fe5bfae3140cea6cafa1c39e7e35a5b3c661";
  };

  nativeBuildInputs = [ lzip texinfo ];
  buildInputs = [ lzlib ];
```

File: pkgs/tools/compression/plzip/default.nix [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+  enableParallelBuilding = true;
+
+  meta = with lib; {
+    homepage = "https://www.nongnu.org/lzip/plzip.html";
+    description =
+      "A massively parallel lossless data compressor based on the lzlib compression library";
```
Comment (SuperSandro2000) [inline pkgs/tools/compression/plzip/default.nix:21]:
```suggestion
    description = "A massively parallel lossless data compressor based on the lzlib compression library";
```

**Pull request merged**